pytorch_lightning/utilities/fetching.py

```diff
@@ -54,15 +71,32 @@ def __init__(
         self.dataloader: Optional[Iterable] = None
         self.dataloader_iter: Optional[Iterator] = None
 
+        self.stage: Optional[str]
```

Comment:
nit in L66: i'm curious why we need to validate that `prefetch_batches` is an int given the typehint specifies this

Reply:
Removed !